javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,138 @@
+    if(command.includes("Remove") && command.includes("from my todo")) {
+        let item = command.replace("Remove ", "");
+        let deleteTodo = item.replace(" from my todo", "");
+        const itemIndex = myTodo.indexOf(deleteTodo);
+        myTodo.splice(itemIndex, 1);
+        response = `Removed ${deleteTodo} from your todo`;
+        console.log(myTodo);
+    }
+    
+    // What is on my todo? - should respond with the todos. Fx you have 2 todos - fishing and singing in the shower
+    if(command.includes("What is on my todo")) {
+        if(myTodo.length === 1) {
+            response = `you have ${myTodo.length} todo - `;
+        } else {
+            response = `you have ${myTodo.length} todos - `; 
+        }
+        for(let i = 0; i < myTodo.length; i++) {
+            if((i === myTodo.length - 1) && (myTodo.length !== 1)) {
+                response += ` and ${myTodo[i]}`; 
+            } else if(myTodo.length === 1) {
+                response += `${myTodo[i]}`;
+            } else if(i === myTodo.length - 2) {
+                response += `${myTodo[i]}`;
+            } else {
+                response += `${myTodo[i]}, `; 
+            }
+        }
+    }
+
+    // What day is it today? - Should respond with the date in a human readable format.
+    // E.g. if today is 30/8/2019 then it should respond with 30. of August 2019
+    if(command.includes("What day is it today")) {
+        const todaysDate = new Date();
+        const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+        response = `${todaysDate.getDate()}. of ${months[todaysDate.getMonth()]} ${todaysDate.getFullYear()}`;
+    }
+
+    // Should be able to do simple math. fx what is 3 + 3 should respond with 6. Or what is 4 * 12 should respond with 48
+    if(command.includes("what is ")) {
+        const actualTask = command.replace("what is ", "");
+        const firstOperand = parseInt(actualTask);
+        if(actualTask.includes("+")) {
+            const n = actualTask.replace(firstOperand, "");
+            const secondOperand = parseInt(n.replace(" + ", ""));
+            response = firstOperand + secondOperand;
+        } else if(actualTask.includes("-")) {
+            const n = actualTask.replace(firstOperand, "");
+            const secondOperand = parseInt(n.replace(" - ", ""));
+            response = firstOperand - secondOperand;
+        } else if(actualTask.includes("*")) {
+            const n = actualTask.replace(firstOperand, "");
+            const secondOperand = parseInt(n.replace(" * ", ""));
+            response = firstOperand * secondOperand;
+        } else if(actualTask.includes("/")) {
+            const n = actualTask.replace(firstOperand, "");
+            const secondOperand = parseInt(n.replace(" / ", ""));
+            response = firstOperand / secondOperand;
+        }
+    }
```

Comment:
Instead of 'n' try using meaningful variable name.
What you did is correct but maybe you can try as below 

You can also try in the below way which is readable and less complicated
You can also change command to array of words then get numbers and operation like this

else if (command.startsWith("what is")) {
    const commandWords = command.split(" ");
    const number1 = parseInt(commandWords[2]);
    const number2 = parseInt(commandWords[4]);
    const operation = commandWords[3];
    if (number1 && number2) {
      switch (operation) {
        case "+":
          return number1 + number2;
        case "-":
          return number1 - number2;
        case "*":
          return number1 * number2;
        case "/":
          return number1 / number2;
      }

Reply:
Wow! Yes, this is way too short and easy than mine :) Thanks.